Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM node:4.2.4-wheezy
+MAINTAINER Mark Stemm (mstemm@jut.io)
+
+RUN mkdir -p /opt/juttle-engine
+WORKDIR /opt/juttle-engine
+ADD package.json package.json
+RUN npm install
+COPY . .
+
+RUN mkdir -p /opt/juttle-engine/juttles
+COPY examples /opt/juttle-engine/juttles/examples
+
+# Also symlink juttle and juttle-engine so they can be run directly
+# without specifying a path.
+RUN ln -s /opt/juttle-engine/bin/juttle /usr/local/bin/juttle
```

Comment:
You should be able to run `npm link` to accomplish this same thing but without having to explicitly enumerate the scripts.